components/brave_vpn/renderer/android/vpn_render_frame_observer_browsertest.cc

```diff
@@ -1,7 +1,7 @@
 /* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
the right place for this file is somewhere around `//brave/browser` because of `chrome/common` include.

Reply:
yes, I understand this was not the ideal place for it, but right now I'm just trying to find some quick fixes. I'll take a look at content isolated_world_ids

Reply:
updated